### ISSUE TITLE
Support for RouteDomain "parent" property

### ISSUE THREAD

**Issue by daniesch** [F5Networks/f5-declarative-onboarding#143]
Declarative Onboarding (DO): Support for RouteDomain "parent" property
 
1.      Describe the new feature in as much detail as possible.
 
Our customer would like Declarative Onboarding (DO) for the basic configuring / onboarding of BIG-IP. Using DO, configuring the "parent" property for Route Domains appears to be not possible at present according to https://clouddocs.f5.com/products/extensions/f5-declarative-onboarding/latest/schema-reference.html#routedomain.
 
It would be perfect if Declarative Onboarding will offer configuration / declaration options for that Route Domain property, as shown in the sample BIG-IP configuration for Route Domain "rd2" with "parent" property set to non-default Route Domain "rd1" below:
 ```
net route-domain 0 {
    id 0
    vlans {
        http-tunnel
        socks-tunnel
    }
}
net route-domain rd1 {
    id 1
    routing-protocol {
        BGP
        BFD
    }
    vlans {
        vlan1
    }
}
net route-domain rd2 {
    id 2
    parent rd1
    vlans {
        vlan2
    }
}
```

**Comment by dstokesf5** [comment]
This is being tracked internally as AUTOTOOL-1532.

**Comment by daniesch** [comment]
@dstokesf5 do we have an estimate when it is fixed?

**Comment by daniesch** [comment]
@dstokesf5 any feedback?

**Comment by dstokesf5** [comment]
There are no updates on this issue at this time. It is currently marked as a low priority item. Re-using properties from another profile is convenient, but should not be a deployment blocker. We are currently focusing on issue that are blocking customers.

**Comment by dstokesf5** [comment]
This issue was resolved as part of the 1.17.0 release.
